string[] joint_names
int32[] command_id
int32[] direction
